src/app/shared/stache/modules/image/image.component.html

```diff
@@ -1 +1,3 @@
-<img class="stache-image" [src]="imageSource" />
+<div class="stache-image-container">
```

Comment:
I understand the thinking here, but I'd prefer to make the "wrapping" class the same as the name of the component (`selector: 'stache-image'`). We could copy Bootstrap's naming convention and use `-object` for the image tag (see [`<img class="media-object">`](http://getbootstrap.com/components/#media)).

```
<div class="stache-image">
  <img class="stache-image-object" />
</div>
```